src/components/Card/Chart/TodayPostCountCard.jsx

```diff
@@ -35,6 +35,9 @@ const TodayPostCountCard = ({ keywordId }) => {
           {noDiff && <EndashIcon className="size-90" />}
           {greaterThanYesterday && <UpwardArrowIcon className="size-90" />}
           {lessThanYesterday && <DownwardArrowIcon className="size-90" />}
+          <p className="text-50 justify-center items-center pt-8">
+            {!noDiff && chartData.diffPostCount}
+          </p>
```

Comment:
```suggestion
          <p className="text-50 justify-center items-center pt-8">
            {isDiff && chartData.diffPostCount}
          </p>
```
변수를 이중부정으로 사용하는 대신, 긍정으로 바꿔보면 어떨까요?

현재 변수는 간단한 의미라 혼란스럽지 않지만, 조금만 변수명이 복잡해지면 가독성이 좋지 않을 수도 있겠다는 의견이에요. (예: `!isNotSuggested`) [참고](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

검토해주시고 저희가 정리 중인 [변수 네이밍 컨벤션](https://www.notion.so/13054df9604380009412f6ae8d5d6f38?pvs=4#edee228876364886b6f8998b8a0f8f3d)에도 업데이트 해봐도 좋겠네요!

Reply:
네네 좋아요! 수정 이전엔 차이가 없는 경우에 대한 로직만 있었는데, 어제와 오늘의 게시물 수에 대한 부분을 추가하면서 기존 변수를 사용하게 되면서 이중 부정이 되었네요..! 
노션 페이지에도 추가해놓겠습니다. 해당 부분 놓치지 않고 알려주셔서 감사해요!